### ISSUE TITLE
Meta Learning with Temporal Convolutions

### ISSUE THREAD

**Issue by rbharath** [deepchem/deepchem#664]
https://arxiv.org/abs/1707.03141

Would love to see an implementation contributed to DeepChem. Wonder how it would compare to IterRef LSTMs

**Comment by patrickhop** [comment]
Would love to see this as well. Did the authors release code?

**Comment by rbharath** [comment]
I don't believe so. It doesn't look like it would be terribly complicated to implement with TensorGraph or PyTorch though.

PRs welcome from anyone interested!

**Comment by rbharath** [comment]
Opening this issue out for contributions. Adding this implementation would be a good intermediate level contribution, especially for those with prior deep learning expertise.

**Comment by HridayM25** [comment]
Hey @rbharath is this still a desired feature to be added?

**Comment by rbharath** [comment]
Closing this old issue as probably out of date
